wire-ios-sync-engine/Source/Data Model/Conversation+MessageDestructionTimeout.swift

```diff
@@ -93,6 +93,14 @@ private enum MessageDestructionTimeoutRequestFactory {
         guard let identifier = conversation.remoteIdentifier?.transportString()
         else { fatal("conversation inserted on backend") }
 
+        let path: String
+        if apiVersion < .v8 {
+            path = "/conversations/\(identifier)/message-timer"
+        } else {
+            guard let domain = conversation.domain else { fatal("conversation has no domain") } // TODO: unit test?
```

Comment:
I would suggest to fallback to the local domain, which there should be (even though it's optional 🙃).

Reply:
I wouldn't crash the app if there was no domain

Reply:
done